src/providers/index.tsx

```
@@ -1,23 +1,27 @@
+]
+
+export function RootProvider({ children }: ProviderProps) {
 	return (
 		<>
-			<CookiesProvider>
-				<ReactQueryProvider>{children}</ReactQueryProvider>
-			</CookiesProvider>
-
-			<TanStackRouterDevtools position="bottom-left" />
-
+      {providers.reduceRight((acc, Provider) => <Provider>{acc}</Provider>, children)}
```

Biome (2.4.4)

[error] Missing key property for this element in iterable.

(lint/correctness/useJsxKeyInIterable)